### ISSUE TITLE
issue #1693 looks like its broken again in 1.9.0

### ISSUE THREAD

**Issue by umbalem** [kenwheeler/slick#3612]
https://github.com/kenwheeler/slick/issues/1693

(not sure how to correctly format this.. sorry)

Behavior described in issue #1693 seems to be happening to me in 1.9.0

Filtering breaks on mobile breakpoints. It works the first time, but filtering again no items are shown. It works fine on full screen.

Filter works after removing the responsive table init.
I have reverted back to 1.5.9 and everything works.

**Comment by pnmcosta** [comment]
Yup, I can confirm, have the same issue, change https://s.codepen.io/pnmcosta/debug/EdJNVJ/PBkNWNjRnKJM to mobile responsive and you can reproduce it.

**Comment by mised** [comment]
I can confirm too. Have some1 any hint?

**Comment by jeffreyducharme** [comment]
Still seeing this issue :(

**Comment by pnmcosta** [comment]
I delved, unfortunately couldn't figure out why, code seems ok... Published the site (https://www.zximenes.com/) anyway, hopefully it will be fixed soon.

**Comment by muster-mark** [comment]
Also been having this problem for a while, meaning I've got to stick to 1.5.9.

**Comment by oilvier** [comment]
Hi,

I can't explain why, but it seems to be linked with the `rows` option.
If you set it to `rows: 0` (it's what you need in most of the cases anyway) the problem disappears.

It could be a sufficient fix until a proper solution is found.

**Comment by pnmcosta** [comment]
@oilvier thanks, I can confirm setting `rows:0` to the breakpoints fixes it.

Great spot, thanks :)

**Comment by prefacestudios** [comment]
`rows: 0` added to each *breakpoint*, not just on the main `.slick({` call worked for me. This fix should be added to the documentation, along with an example of how to filter by a CSS class.

**Comment by patrick-wc** [comment]
rows: 0 added to each breakpoint, not just on the main .slick({ call worked for me. This fix should be added to the documentation, along with an example of how to filter by a CSS class.

**Comment by dimitur2204** [comment]
Incredible spot. Works for 1.8.0 too.

**Comment by xahid07** [comment]
@oilvier Mindblowing solution. Thank you so much.
